README.md

```diff
@@ -376,19 +374,20 @@ you chain more over the same structure. It's therefore recommended that you use
 traverse a large list (10000+ items).
 
 #### ap
-##### `#ap :: Future a b ~> Future a (b -> c) -> Future a c`
+##### `#ap :: Future a (b -> c) ~> Future a b -> Future a c`
```

Comment:
My concern is that someone might see the type signature and assume that Fluture targets an old version of FL. It might be worth mentioning in the description that `fantasy-land/ap` is FL compliant.